.travis.yml

```diff
@@ -0,0 +1,56 @@
+stages:
+  - unit_test
+  - integration_test
+
+language: java
+
+services:
+  - docker
+
+jdk:
+  - openjdk11
+
+os:
```

Comment:
Can you put OS at the top please? I like to start with the more general stuff and then move down to more specific.